src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -643,4 +643,37 @@
   <data name="PX1072Title_CreateSharedGraphVariable" xml:space="preserve">
     <value>Creating a graph for a BQL query is a resource consuming operation. To decrease the number of these operations, create a variable with a shared graph that can be reused by BQL queries.</value>
   </data>
+  <data name="PX1062MessageFormat" xml:space="preserve">
+    <value>Graphs and graph extensions cannot declare static {0}.</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Fields" xml:space="preserve">
+    <value>mutable fields</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Properties" xml:space="preserve">
+    <value>mutable properties</value>
+  </data>
+  <data name="PX1062FixMakeReadOnlyFormat" xml:space="preserve">
+    <value>Make the {0} read-only</value>
+  </data>
+  <data name="PX1062FixFormatArg_Field" xml:space="preserve">
+    <value>field</value>
+  </data>
+  <data name="PX1062FixFormatArg_Property" xml:space="preserve">
+    <value>property</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Views" xml:space="preserve">
+    <value>views</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Actions" xml:space="preserve">
+    <value>actions</value>
+  </data>
+  <data name="PX1062FixMakeNonStaticFormat" xml:space="preserve">
+    <value>Remove static from the {0}</value>
```

Comment:
@EAndrosova  please review the code fix message: `Remove static from the {0}`
The possible format arguments (stored here in the resources):
- field
- property
- view
- action

Reply:
@SENya1990  I think this sentence should be a bit more specific: "Remove the static modifier from the {0}"
Also, can this message specify the exact name of the field? For example, _Remove the static modifier for the InvoiceNbr field._

Reply:
After discussion it was decided to follow MS style for diagnostic messages:

-     No dots for single sentences
-     Dots at the end of all sentences if there are two or more sentences

Reply:
done